core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -756,4 +762,22 @@ public final void setIdleTimeout(final int idleTimeoutMillis) {
             this.idleTimeoutMillis = idleTimeoutMillis;
         }
     }
+
+    /**
+     * TODO.
+     * 
+     * @return The inactiveLinkTimeout in milliseconds.
+     */
+    public long getInactiveLinkTimeout() {
+        return inactiveLinkTimeout;
+    }
+
+    /**
+     * TODO.
+     * 
+     * @param inactiveLinkTimeout The inactiveLinkTimeout in milliseconds.
+     */
+    public void setInactiveLinkTimeout(final long inactiveLinkTimeout) {
```

Comment:
final
should we enforce a minimum value, e.g. 0 or 60 seconds? does it make sense to measure the timeout in milliseconds? Seconds should do the trick FMPOV

Reply:
> should we enforce a minimum value, e.g. 0 or 60 seconds? 

I already enforce a minimum value of 0. Even though I don't think that very small values are useful.

> does it make sense to measure the timeout in milliseconds? Seconds should do the trick FMPOV

I used milliseconds to use the same unit as the other time related properties in `ClientConfigProperties`, hoping that this would be less confusing and error prone in the configuration and it is the unit the timer uses, so i do not need to do calculations between the unit of the timeout and the unit of the implementation (Vertx timer, and timestamps). But on the other hand I expect timeouts to be rather hours than fractions of a second and values in milliseconds get really big. So I can happily change that.

Reply:
that was my concern as well. I also expect the timeout to be rather hours than seconds. So why not use minutes?

Reply:
For production use minutes are probably a good choice. But for integration tests even one one minute is too long FMPOV.

Reply:
Note: now using one property for the amount of time and one for the time unit.